science-platform-int/templates/authnz-application.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: auth-int
```

Comment:
Can we make this auth instead? That will require a redeploy in int, but it gets rid of a configuration difference.